Problem4/src/problem4/Ball.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package problem4;
+
+import java.awt.Color;
+import java.awt.Graphics;
+import java.util.Random;
+
+/**
+ *Creates a ball
+ * @author bachle
+ */
+public class Ball {
```

Comment:
Your helper Ball class should be implemented using OOP best practices.  Member variables should be private with appropriate getters and setters.  You should also be using a constructor to initialize the ball’s radius, positioning and movement speed.  This would have allowed you to easily vary the colour and radius of each ball (if you wanted).